src/test/java/org/jsoup/parser/TokenQueueTest.java

```diff
@@ -1,74 +1,98 @@
 package org.jsoup.parser;
 
+import junitparams.JUnitParamsRunner;
+import junitparams.Parameters;
+import org.jsoup.Jsoup;
 import org.junit.Test;
-import static org.junit.Assert.*;
+import org.junit.runner.RunWith;
+
+import static org.junit.Assert.assertEquals;
 
 /**
  * Token queue tests.
  */
-public class TokenQueueTest {
-    @Test public void chompBalanced() {
-        TokenQueue tq = new TokenQueue(":contains(one (two) three) four");
-        String pre = tq.consumeTo("(");
-        String guts = tq.chompBalanced('(', ')');
-        String remainder = tq.remainder();
-
-        assertEquals(":contains", pre);
-        assertEquals("one (two) three", guts);
-        assertEquals(" four", remainder);
-    }
-    
-    @Test public void chompEscapedBalanced() {
-        TokenQueue tq = new TokenQueue(":contains(one (two) \\( \\) \\) three) four");
-        String pre = tq.consumeTo("(");
-        String guts = tq.chompBalanced('(', ')');
-        String remainder = tq.remainder();
-
-        assertEquals(":contains", pre);
-        assertEquals("one (two) \\( \\) \\) three", guts);
-        assertEquals("one (two) ( ) ) three", TokenQueue.unescape(guts));
-        assertEquals(" four", remainder);
-    }
-
-    @Test public void chompBalancedMatchesAsMuchAsPossible() {
-        TokenQueue tq = new TokenQueue("unbalanced(something(or another)) else");
-        tq.consumeTo("(");
-        String match = tq.chompBalanced('(', ')');
-        assertEquals("something(or another)", match);
-    }
-    
-    @Test public void unescape() {
-        assertEquals("one ( ) \\", TokenQueue.unescape("one \\( \\) \\\\"));
-    }
-    
-    @Test public void chompToIgnoreCase() {
-        String t = "<textarea>one < two </TEXTarea>";
-        TokenQueue tq = new TokenQueue(t);
-        String data = tq.chompToIgnoreCase("</textarea");
-        assertEquals("<textarea>one < two ", data);
-        
-        tq = new TokenQueue("<textarea> one two < three </oops>");
-        data = tq.chompToIgnoreCase("</textarea");
-        assertEquals("<textarea> one two < three </oops>", data);
-    }
-
-    @Test public void addFirst() {
-        TokenQueue tq = new TokenQueue("One Two");
-        tq.consumeWord();
-        tq.addFirst("Three");
-        assertEquals("Three Two", tq.remainder());
-    }
-    
-    
-    @Test 
-    public void consumeToIgnoreSecondCallTest(){
-		String t = "<textarea>one < two </TEXTarea> third </TEXTarea>";
-		TokenQueue tq = new TokenQueue(t);
-		String data = tq.chompToIgnoreCase("</textarea>");
-		assertEquals("<textarea>one < two ", data);
-		
-		data = tq.chompToIgnoreCase("</textarea>");
-		assertEquals(" third ", data);
-    }
-    
+@RunWith(JUnitParamsRunner.class) public class TokenQueueTest {
+  @Test public void chompBalanced() {
```

Comment:
The rest of the code base uses 4 spaces for indentation. Did you change it here on purpose?

Reply:
No. Sorry I didn't know if you had strict guidelines for style. But I've been trying to move to Google style guide and used it here too. Would you like me to submit a different pull request with the original style.?